Documentation/ApiOverview/ContentSecurityPolicy/Index.rst

```diff
@@ -159,7 +159,12 @@ or styles are blocked by the browser.
 
 The nonce is applied automatically, when scripts or styles are defined with the
 TYPO3 API, like TypoScript (:typoscript:`page.includeJS`, etc.) or the
-:ref:`asset collector <assets>`.
+:ref:`asset collector <assets>`. This only refers to referenced files
+(via :html:`src` and :html:`href` attributes) and not inline scripts
+or stylesheets. For those, you should either use the PHP/Fluid approach
+as listed below, or use TypoScript only for passing DOM attributes
```

Comment:
I assume with "DOM attributes" you mean "data-" attributes?

Reply:
It doesn't necessarily mean "data-" attributes, depending on what someone wants to achieve they might use normal attributes i.e. to meta tags or so. So I wanted to be broader with that.

But if you feel better about writing "data-" here I can agree, please feel free to commit that suggestion directly :) Thanks!

Reply:
Would you have time/motivation to work on this? Else I think it's an improvement over the current situation and we would need fine-tuning in a follow-up...?

Reply:
I would vote for merging it like this, when someone has time he can make a follow up

Reply:
Ok, will merge for now and see if maybe this creates additional feedback later, thanks!